adback-solution-to-adblock/ad-back.php

```diff
@@ -48,6 +48,7 @@ function activate_ad_back($networkwide) {
     require_once plugin_dir_path( __FILE__ ) . 'includes/class-ad-back-activator.php';
     require_once plugin_dir_path( __FILE__ ) . 'includes/class-ad-back-updator.php';
     require_once plugin_dir_path( __FILE__ ) . 'includes/class-ad-back-get.php';
+    Ad_Back_Updator::onUpdateFromOldVersion();
```

Comment:
you should also launch this method if there is a plugin update I think

Reply:
ok done.